manifests/n/nektos/act/0.2.35/nektos.act.installer.yaml

```
@@ -0,0 +1,23 @@
+PackageIdentifier: nektos.act
+PackageVersion: 0.2.35
+Installers:
+- Architecture: x64
+  ReleaseDate: 2023-01-01
+  InstallerType: zip
+  InstallerUrl: https://github.com/nektos/act/releases/download/v0.2.35/act_Windows_x86_64.zip
+  InstallerSha256: 150162997bd4c4fee84d1f1384975c0ebb06490f0d2bb6cdb96129112bf8b3ad
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: act.exe
+    PortableCommandAlias: act
+- Architecture: x86
+  ReleaseDate: 2023-01-01
+  InstallerType: zip
+  InstallerUrl: https://github.com/nektos/act/releases/download/v0.2.35/act_Windows_i386.zip
+  InstallerSha256: 67760ac46cc196681b31d23faf8a16995d6337d5ad229af737d7b2aeb08e1e09
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: act.exe
+    PortableCommandAlias: act
+ManifestType: installer
+ManifestVersion: 1.2.0
```

```suggestion
ManifestVersion: 1.4.0
```
Zip is only supported in Manifest Version 1.4.0